packages/react-components/react-search/src/components/SearchBox/useSearchBox.tsx

```diff
@@ -55,15 +80,28 @@ export const useSearchBox_unstable = (props: SearchBoxProps, ref: React.Ref<HTML
         children: <DismissRegular />,
         role: 'button',
         'aria-label': 'clear',
+        tabIndex: -1,
       },
       required: true,
     }),
-    contentAfter: resolveShorthand(contentAfter, { required: true }),
+    contentAfter: resolveShorthand(contentAfter, {
+      required: true,
+    }),
 
     disabled,
+    focused,
     size,
   };
 
+  const searchBoxRoot = state.root.root as InputState['root'];
+  const onRootFocus = useEventCallback(mergeCallbacks(searchBoxRoot.onFocus, onFocus));
```

Comment:
On L99 you're null checking `searchBoxRoot` but accessing `searchBoxRoot.onFocus/onBlur` here. I think you need to move L97-98 inside the conditional on L99.

Reply:
Thinking about this some more, what is a case where `searchBoxRoot` would not exist?